routes/movies.js

```diff
@@ -0,0 +1,22 @@
+const router = require("express").Router();
+const Movie = require("../models/Movie");
+
+// Iteration 8
+
+router.get('/new', (req, res, next) => {
+  res.render('movies/new.hbs');
+  // Get all celebrities and pass them to view
+});
```

Comment:
The comment here does not match the route. This one is to create a new movie. For this, you will also need to get the list of celebrities from the database and pass it to the hbs file. This way the list of celebrities you choose from when creating a new movie will always come from the database.